flowpm/tfpm.py

```diff
@@ -7,7 +7,9 @@
 import tensorflow as tf
 from flowpm.tfbackground import f1, E, f2, Gf, gf, gf2, D1, D2, D1f
 from flowpm.utils import white_noise, c2r3d, r2c3d, cic_paint, cic_readout
-from flowpm.kernels import fftk, laplace_kernel, gradient_kernel, longrange_kernel
+from flowpm.kernels import fftk, laplace_kernel, gradient_kernel, longrange_kernel  #,  PGD_kernel
+
+from flowpm.pk import pk as pkl
```

Comment:
@EiffL The pgd function defined in tfpm.py returns the 3D power spectrum and uses the pkl to compute it

Reply:
Sure, :-) but the term "pkl" it doesn't mean anything to me here, so not a good name to give to a function. It could be misinterpreted as meaning linear pk